schemas/external/page/page.schema.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
@trieloff - this schema is now in schemas/external/page folder (following standard seen with schemas/external/hal to have folder per common type. @fmeschbe proposed moving this to schemas/commons folder to be even more generic. Please advise if that would be desired location and I'll move it there.

Reply:
I think common sounds right. It's not really an external standard (like HAL that we re-implemented), but very common.